raster/src/main/scala/geotrellis/raster/GridExtent.scala

```diff
@@ -60,13 +64,13 @@ class GridExtent[@specialized(Int, Long) N: Integral](
   * same cellsizes).  The result is a new extent at the same
   * resolution.
   */
-  def combine (that: GridExtent[N]): GridExtent[N] = {
+  def expandToInclude (that: GridExtent[N]): GridExtent[N] = {
```

Comment:
The `combine` function on `Extent` and `RasterExtent` I'd expect to be very heavily used in user code. `expendToInclude` is a way better name but I'm wondering if the frequency with which these are used warrants going through the alias/depricate cycle on these in 3.0 release.